### ISSUE TITLE
CTS indicates full support for entity neighborhood when they are not implemented

### ISSUE THREAD

**Issue by cmgrote** [odpi/egeria#2075]
This output suggests that there is full entity neighborhood support despite the details indicating that graph query support is disabled -- probably a bug, and should be something more like `CONFORMANT_NO_SUPPORT`? 😉 

```json
{
  "id": 22,
  "name": "Entity neighborhood",
  "description": "The technology under test supports the ability to query metadata instances that radiate from a specific entity instance through multiple hops.",
  "documentationURL": "https://egeria.odpi.org/open-metadata-conformance-suite/docs/repository-workbench/profiles/graph-queries/entity-neighborhood",
  "conformanceStatus": "CONFORMANT_FULL_SUPPORT",
  "positiveTestEvidence": [
    {
      "profileId": 4,
      "requirementId": 22,
      "testCaseId": "repository-graph-queries-<null>",
      "testCaseName": "Repository graph query test case",
      "testCaseDescriptionURL": "https://egeria.odpi.org/open-metadata-conformance-suite/docs/repository-workbench/test-cases/repository-graph-queries-test-case.md",
      "testEvidenceType": "DISCOVERED_PROPERTY",
      "propertyName": "Graph query support",
      "propertyValue": "Disabled"
    }
  ]
}
```

**Comment by grahamwallis** [comment]
Hi Chris - nope - this is working as designed. If a repository does not support a profile (or requirement) but behaves correctly in reporting it then the discovered property is set to disabled and the repository is compliant (fully compliant).

**Comment by cmgrote** [comment]
Hmmm...  What's the point of this status, then?

```java
CONFORMANT_NO_SUPPORT        (3, "Conformant with no support",
                                     "The technology provides correctly functioning responses that indicate it has no support for the features in this profile."),
```

**Comment by grahamwallis** [comment]
Good question - and I think I need to change them to that :-)

**Comment by cmgrote** [comment]
Resolved by #2093
